### PR TITLE
Tom branch

### PR DESCRIPTION
**Pull request by Tom-Fynes**

### CODE REVIEW

File: homework/grade2/Tom-grade-2.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# homework for grade 2
```
Comment (lorilew) [inline homework/grade2/Tom-grade-2.py:1]: I have the feeling you really like the `input` method!

File: homework/grade2/Tom-grade-2.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    check = input('Enter a number or string: ')
+
+    if check.isalpha():
+        print('{} is a string'.format(check))
+    elif check.isnumeric():
+        print('{} is a number'.format(str(check)))
+    else:
+        print('{} characters and numbers'.format(check))
+
+def doathing(number):
```
Comment (lorilew) [inline homework/grade2/Tom-grade-2.py:13]: Not taught you this yet, but in python3 it's good practice to follow pep8 coding standards. So function names use underscores, eg: `do_a_thing` and class names are Camel case, eg `ClassThatDoesStuff`.

File: homework/grade2/Tom-grade-2.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        if len(number) < 50:
+            print('Lets count down')
+            while int(control) > 0:
+                print(control)
+                control = int(control) - 1
+    else:
+        print('You must enter a number')
+        doathing(input('Please enter a number less then 100: '))
+
+def lastthing(somestring):
```
Comment (lorilew) [inline homework/grade2/Tom-grade-2.py:27]: For good practice can you add pydocs (Grade 3) to this? It will make it easier to understand.

**Review by lorilew — CHANGES_REQUESTED** [review]

File: homework/grade2/Tom-grade-2.py (hunk 3 above)
Comment (lorilew) [reply homework/grade2/Tom-grade-2.py:27]: https://realpython.com/documenting-python-code/

**Comment by Tom-Fynes** [comment]
Forgot this was even still open haha, closing this off as no more python training :slightly_frowning_face: :stuck_out_tongue:

**Pull request closed without merging**